WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalSuccess.swift

```diff
@@ -47,7 +47,7 @@ final class CardPresentModalSuccess: CardPresentPaymentsModalViewModel {
 
     func didTapSecondaryButton(in viewController: UIViewController?) {
         emailReceiptAction()
-        viewController?.dismiss(animated: true)
+        viewController?.dismiss(animated: false)
```

Comment:
Probably not strictly necessary, but it seemed to make the presentation of the email controller less janky in my tests 🤷🏼